zhaquirks/philips/__init__.py

```diff
@@ -152,15 +155,35 @@ class PhilipsRemoteCluster(CustomCluster):
                 "param2": t.uint24_t,
                 "press_type": t.uint8_t,
                 "param4": t.uint8_t,
-                "param5": t.uint8_t,
-                "param6": t.uint8_t,
```

Comment:
What was the reason again that `param6` is removed here?

Reply:
I think these params were added at a time when it wasn't yet fully understood what these bytes in the message actually mean. By now we know that param5 and param6 (previously both uint8) are actually one single uint16, representing the time a button is held down during a long press event. This code removes replaces param5 (uint**8**) and param6 (uint**8**) with a single param5 (uint**16**). I was a bit conflicted about this too. If anyone realized that at least one of them represent long press duration, they would probably have seen the value of param6 increase - maybe not pressing long enough for the byte in param5 to change. Given this, there's maybe a case to be made to keep the name param6 have it be a uint16, and remove and skip param5.

Reply:
Ah, thanks for the explanation. I think leaving it like this (`param5`) should be fine.